### DISCUSSION TITLE
Deploying latest version of OHIF DICOM Viewer

### DISCUSSION THREAD

**Discussion by lijinjoseji** [microsoft/dicom-server#2281] [General]
Hello Team,

Do we have a documentation to follow to deploy latest version of OHIF Viewer (replacing the existing deployed version in Image Server)? As we are already setup the Image server, just want to replace it without making any impact.

Any help will be greatly appreciated.

Thanks in advance
Lijin

**Comment by smithago** [comment]
https://github.com/microsoft/dicom-server/blob/main/src/Microsoft.Health.Dicom.Web/wwwroot/index.html#L23

You can replace it there with whichever version works for you.

Note: We do not manage or own OHIF.

**Reply by DevOriginalWang** [reply]
If i deployed via Azure, is there anywhere to change it in Azure?
